packages/@webex/plugin-cc/test/unit/spec/services/core/aqm-reqs.ts

```
@@ -0,0 +1,570 @@
+      postMessage: jest.fn(),
+      onmessage: jest.fn(),
+    };
+
+    global.Worker = jest.fn(() => mockWorker) as any;
+
+    webSocketManagerInstance = new WebSocketManager({
+      webex: {} as any,
+    }) as jest.Mocked<WebSocketManager>;
+
+    // Mock the on method to handle event listeners
+    const eventListeners: { [key: string]: Function[] } = {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)